apps/admin/frontend/src/components/export_election_ballot_package_modal_button.test.tsx

```diff
@@ -94,8 +112,15 @@ test('Button renders properly when not clicked', () => {
 
 test('Modal renders insert usb screen appropriately', async () => {
   const usbStatuses: UsbDriveStatus[] = ['absent', 'ejected', 'bad_format'];
+  // Need 1 ApiMock per test, so we recreate the global-scoped mock to clobber
+  // the expectations we created in `beforeAll()`.
+  // TODO add an explicit 'clear expectations' function
```

Comment:
We actually used to provide a reset function, but it didn't always suit our needs so we intentionally shifted to a "recreate the mock for each test" approach: https://github.com/votingworks/vxsuite/pull/2986#discussion_r1098921442

Reply:
Probably should update this file to just move the mock creation out of `beforeAll` and accept some repeated boilerplate

Reply:
see discussion below; https://github.com/votingworks/vxsuite/pull/3147/commits/8cbf449e251c2e618419cd0b359f3bcc10b12c2f